CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ Changes by Version
 
 ##### Breaking Changes
```

Comment:
I am guessing this is technically a non-breaking change. It will become breaking once we start defaulting to _v2